build.gradle

```diff
@@ -7,20 +7,19 @@ if (hasProperty("release")) {
 	version = [ "git", "describe", "--match", "[0-9]*", "--abbrev=0"].execute().text.trim() + "-SNAPSHOT"
 }
 
-task version << {
+task version {
```

Comment:
the println should be wrapped in a `doLast` block, so that it's not executed configuration time ( `<<` did the same)